### PR TITLE
Update links to 1.0.0_1 installers

### PR DESCRIPTION
**Pull request by hoechenberger**
To be merged once the new installers have been released.

See https://github.com/mne-tools/mne-installers/pull/111

### CODE REVIEW

File: doc/install/installers_platform_selector.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h5 class="m-0 font-weight-bold">
     <div id="collapseWindows" class="collapse" aria-labelledby="windows" data-parent="#downloadAccordion">
       <div class="card-body">
         <a class="btn btn-primary font-weight-bold shadow-sm mt-3 mb-3" role="button"
-          href="https://github.com/mne-tools/mne-installers/releases/download/v1.0.0/MNE-Python-1.0.0_0-Windows.exe">
+          href="https://github.com/mne-tools/mne-installers/releases/download/v1.0.0-post1/MNE-Python-1.0.0_1-Windows.exe">
```
Comment (larsoner) [inline doc/install/installers_platform_selector.html:72]: https://github.com/mne-tools/mne-installers/releases/download/v1.0.0-post1/MNE-Python-1.0.0_1-Windows.exe is dead

Comment (larsoner) [reply doc/install/installers_platform_selector.html:72]: ... ahh, release not complete yet. Let me finish testing Windows now!

**Comment by hoechenberger** [comment]
@larsoner Can you merge please? I just published the new installers.

I don't have permission here to merge:
Screen Shot 2022-03-31 at 19 12 52

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Comment by hoechenberger** [comment]
@larsoner Could you backport if you have a minute? I'm currently on the road…

**Comment by larsoner** [comment]
Done!

**Pull request merged**